### ISSUE TITLE
ECR: Not supported because n parameter of list_images to high

### ISSUE THREAD

**Issue by theCapypara** [andrey-pohilko/registry-cli#79]
Right now ECR registries are not supported, because the "n" parameter of the image listing request is too high.

This is the error message:
`{"errors":[{"code":"UNSUPPORTED","message":"Invalid parameter at 'maxResults' failed to satisfy constraint: 'Member must have value less than or equal to 1000'"}]}`

**Comment by andrey-pohilko** [comment]
ECR Registry doesn't have any clean up functionality?

**Comment by theCapypara** [comment]
Yes I also noticed this after manually changing the limit (although I found it weird that it is not explained in the AWS docs that deleting repositories via the Docker API does not work, but of course that's no issue of registry-cli).

I guess it would still be nice to use registry-cli to list images in ECR repositories though.

**Comment by andrey-pohilko** [comment]
Please correct me if I'm wrong. Does this message from AWS mean that you have more than 1000 images in your repository there?

**Comment by theCapypara** [comment]
No, ECR complains that the limit parameter used is too high. Lowering it below 1000 works. We only have a couple images.
